scripts/ci/coverity.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname "$0")"/../..
+./bootstrap
+./configure
```

Comment:
Adding `--enable-debug=full` option would ensure that all debug code is also included.

Reply:
Ok, I'll add this.